pkg/operator/target_config_reconciler.go

```diff
@@ -1066,6 +1101,10 @@ func relieveAndMigrateProfile(profileCustomizations *deschedulerv1.ProfileCustom
 		}
 	}
 
+	lowThreshold, highThreshold, err := getRelieveAndMigrateThresholds(profileCustomizations, args.UseDeviationThresholds)
```

Comment:
You could probably keep the original `getLowNodeUtilizationThresholds` around, set both `lowThreshold` and `highThreshold` to the default asymmetric `0` and `10` and invoke `getLowNodeUtilizationThresholds` only when `profileCustomizations != nil`.

Reply:
`getRelieveAndMigrateThresholds` is internally doing that